src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php

```diff
@@ -68,6 +70,32 @@ protected function addRepository(ContainerBuilder $container, MetadataInterface
      */
     protected function addDefaultForm(ContainerBuilder $container, MetadataInterface $metadata)
     {
+        $options = array_merge(
+            [
+                'default_parent_path' => null,
+                'autocreate' => false,
+            ],
+            $metadata->hasParameter('options') ? $metadata->getParameter('options') : []
+        );
```

Comment:
Would be nice to have options handled in some way that provided defaults and disallowed specification of non-valid  keys.